DESCRIPTION

```diff
@@ -27,3 +27,4 @@ RoxygenNote: 6.1.1
 VignetteBuilder: knitr 
 URL: https://github.com/gitter-lab/LPWC
 BugReports: https://github.com/gitter-lab/LPWC/issues
+Imports: nleqslv
```

Comment:
For my later reference, this is a good explanation of Depends versus Imports https://kbroman.org/pkg_primer/pages/depends.html